src/main/user/user-request-authorizer.ts

```diff
@@ -1,24 +1,33 @@
 import { getTokenDetails } from "./user-resolver";
+import { isUserAuthorized } from "../role/roles-based-authorizer";
 
 export const ERROR_TOKEN_MISSING = {
   error: "Bearer token missing",
-  message: "You are not authorized to access this resource",
+  message: "You are not authorised to access this resource",
   status: 401,
 };
 export const ERROR_UNAUTHORISED_ROLE = {
   error: "Unauthorised role",
-  message: "You are not authorized to access this resource",
+  message: "You are not authorised to access this resource",
   status: 403,
 };
 export const ERROR_UNAUTHORISED_USER_ID = {
   error: "Unauthorised user",
-  message: "You are not authorized to access this resource",
+  message: "You are not authorised to access this resource",
   status: 403,
 };
 
 export const COOKIE_ACCESS_TOKEN = "accessToken";
 export const AUTHORIZATION = "Authorization";
 
+const authorizeRoles = (user) => new Promise((resolve, reject) => {
+  if (!isUserAuthorized(user)) {
+    reject(ERROR_UNAUTHORISED_ROLE);
```

Comment:
It looks like you changed the messages to use `authorised` but left the variables and filenames to use `authorized`  This is the first time a constant is defined and it is interesting to see you adopt `unauthorised`

Reply:
I'll fix this up. Convention should be that variable names use American English (i.e. `authorized`) but any values to be displayed on screen, such as error messages, should be in British English (i.e. `authorised`).